CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+### Go version
+
```

Comment:
That's not right, it works for me with 1.19, other confluent repos run newer than 1.19 and this also works